dpdbaltics.php

```diff
@@ -1162,6 +1162,10 @@ public function printMultipleLabels($shipmentIds)
             $shipment = new DPDShipment($shipmentId);
             $plNumbers[] = $shipment->pl_number;
             if ($isAutomated) {
+                if(!$shipment->return_pl_number) {
+                    $shipmentService = $this->getModuleContainer('invertus.dpdbaltics.service.shipment_service');
+                    $shipment = $shipmentService->createReturnServiceShipment(Config::RETURN_TEMPLATE_DEFAULT_ID, $shipment->id_order);
+                }
```

Comment:
can add `continue;` as adding null value to array requires to clean them up later on.